wandesky/ride-my-way-2
# [Feature #158783408] refactor code by using blueprints

#### What does this PR do?
The PR introduces Blueprints. The use of blueprints makes routing 

#### Description of Task to be completed?
Refactors code to implement use of blueprints

#### How should this be manually tested?
git clone https://github.com/wandesky/ride-my-way-2.git
git checkout ch-refactor-code-to-use-blueprint-158783408

open terminal and run `python run.py`
open the routes in terminal e.g.
`http://127.0.0.1:5000/api/v1/rides`

should return a dictionary consisting of different rides

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[#158717591](https://www.pivotaltracker.com/story/show/158717591)
